### DISCUSSION TITLE
Somebody please help me !! I don't know where else to go.

### DISCUSSION THREAD

**Discussion by GetOneUpya** [community/community#78247] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Let me start this by saying I myself am technologically retarded. All of the information in this post, the limited knowledge I do have I've gained over the last 6 weeks trying to solve this, and my comments/views/opinions on things could be technically wrong but if you contact me directly I would be happy to answer any more questions and provide whatever I need to to get to the bottom of this.

For the last 6 or so weeks somebody I know personally has been fucking with mine and my partners relationship and lives. It's tearing us apart and driving me insane. 

- All of our devices and accounts were taken over and enrolled into enterprise/work accounts and or put into developer mode. 
- This person has gained access and admin control over almost every aspect of our lives from our devices to our home networks, bank accounts, social media, everything and is intentionally doing it to harm us.
- I'm certain it's a targeted attack by somebody we know personally
- As far as I can tell they are using .Net framework, google workspace, a Microsoft/windows server, via Ethernet, python script, and probably slot of other stuff I don't understand.
- It's being done remotely thru her PC as her username/email address is the admin for the certificate editor, but they've gained access to everything.

I want to find out who this is so I can put and end to it before it further ruins our lives, drives me mad, or ruins the relationship I have with the woman of my dreams.

Were good people, help anyone we can whenever we can, we volunteer for a charity that she works her ass off for 6 days a week.

Please help us.
![Uploading IMG20231208002120~2.jpg…]()
![Uploading Screenshot_2023-12-08-12-41-10-59_61c78dc80ee02b53007c815fefe993e3.jpg…]()

**Comment by git261** [comment]
idk how this got in my internet search, but it did... I'm very sorry that sounds like a messed up situation. All I can offer is what I would do since I'm no expert either...

FIRST You ought to post to many forums and various subreddits, 
e.g. r/AskReddit/ r/needadvice/ r/Advice/ r/Assistance/ r/techsupport/ r/AskForHelp/

get enough attention to find someone who'll help you who has the knowledge/skill, since you mostly talking 2 the wind here, make more posts each day, if one of them blows up the better your chances someone who can, will be able to help you. 


Next or If all else fails, can also ask a white hacker or find discord groups of experts ask them, for help/advice. (don't trust or give money to anyone tho you aren't sure is legit.)

https://www.kaspersky.com/resource-center/definitions/hacker-hat-types

e.g. I found with a bit of searching... (do your own research into it)
https://www.knowledgehut.com/blog/security/ethical-hacking-forums
https://qr.ae/pKOVbs
https://stefan-p-bargan.medium.com/heres-a-list-of-great-cybersecurity-discord-servers-cf7aec7d18bb
https://top.gg/servers/tag/hacking
https://disboard.org/servers/tag/ethical-hacking
https://disboard.org/servers/tag/white-hat
https://discadia.com/?q=whitehat

**Comment by evilgensec** [comment]
Hello. Sorry to hear what you have been through lately. While I cannot guide you to the fullest on how to disclose someone's identity what I can do is help you verify / make an estimation on who the culprit might be.

Canary tokens are unique, trackable identifiers—such as URLs, email addresses, or embedded files—that trigger alerts when accessed, alerting security teams to potential unauthorized access or data leakage. Strategically placed within sensitive files, databases, or embedded in critical applications, these tokens are designed to be covertly monitored. When an attacker accesses a canary token, it sends a signal to a monitoring system, providing details like the attacker’s IP address, user-agent information, and timestamp of access. 

https://www.hackercombat.com/how-to-use-canary-tokens-for-threat-detection/
